enatega-multivendor-app/src/screens/Account/Account.js

```
@@ -71,44 +61,40 @@ function Account(props) {
-    {
-      onCompleted: onCompletedDeactivate,
-      onError: onErrorDeactivate
-    }
-  )
+  const currentTheme = {
+    isRTL: i18n.dir() === 'rtl',
+    ...theme[themeContext.ThemeValue]
+  }
 
+  const [deactivated, { loading: deactivateLoading }] = useMutation(DEACTIVATE, {
+    onCompleted: onCompletedDeactivate,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)